src/constructors/injectGlobal.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import hashStr from '../vendor/glamor/hash'
+import warnOnce from '../vendor/postcss/warn-once'
```

Comment:
You can drop the warning now that we're going directly into 4.0 with this new API.

Reply:
This still needs to be done.